docs/how-to/add-delete-user-entity.md

```diff
@@ -2,16 +2,54 @@
 id: add-delete-user-entity
 title: How To Add and Delete The User Entity
 sidebar_label: How To Add, Delete, and Change The User Entity
-slug: /how-to/add-delete-user-entity
+slug: /user-entity
 pagination_next: getting-started/authentication
 ---
 
-# How To Add, Delete, and Change The User Entity
+# The User Entity
 
-The _User_ entity **is required** in order for you to enable authentication on your service.
+The _User_ entity is essential for the proper functioning of the authentication plugins in Amplication. It **is required** in order for you to enable authentication on your service.
 
 If you choose to [create a service _without_ authentication](/authentication/#how-to-create-a-service-without-authentication) then the _User_ entity will not be created.
 
+## Dependencies Between the User Entity and Authentication Plugins
+
+The authentication plugins, such as the _NestJS Auth Module_ and others, rely on the existence of a _User_ entity to handle user authentication and authorization.
+
+If the _User_ entity is not defined or is missing the required fields, the build process will fail, and the authentication plugins will not function correctly.
+
+It's important to ensure that the _User_ entity is properly defined with the necessary fields before enabling any authentication plugins in your Amplication service.
+
+## Default User Entity Fields
+
+When you create a new _User_ entity in Amplication, it comes with a set of default fields that are necessary for the authentication plugins to work correctly:
+
+- `id`: An automatically created unique identifier of the entity. It is a required and unique field.
```

Comment:
@mulygottlieb - Can you please review the list, and maybe add for the relevant fields for each auth plugins they are required?